plugins/importexport/crossref/classes/form/CrossRefSettingsForm.inc.php

```diff
@@ -140,7 +141,7 @@ function getFormFields() {
 	 * @return boolean
 	 */
 	function isOptional($settingName) {
-		return in_array($settingName, array('username', 'password', 'automaticRegistration'));
+		return in_array($settingName, array('username', 'password', 'automaticRegistration', 'testMode'));
```

Comment:
Happy to see test mode moving to a setting rather than an obscure URL parameter :)